### ISSUE TITLE
docker container waits for a long time, then crashes

### ISSUE THREAD

**Issue by Ark-kun** [github-changelog-generator/github-changelog-generator#717]
```
$ docker run -it --rm -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator --user kubeflow --project pipelines
Using these options:


/usr/local/lib/ruby/2.5.0/delegate.rb:85: [BUG] Segmentation fault at 0x000000000000000c
ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux-musl]

-- Control frame information -----------------------------------------------
c:0014 p:---- s:22160 e:000079 CFUNC  :call
c:0013 p:0087 s:0075 e:000074 METHOD /usr/local/lib/ruby/2.5.0/delegate.rb:85
c:0012 p:0110 s:0066 e:000065 METHOD /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59 [FINISH]
c:0011 p:---- s:0054 e:000053 CFUNC  :call
c:0010 p:0087 s:0049 e:000048 METHOD /usr/local/lib/ruby/2.5.0/delegate.rb:85
c:0009 p:0006 s:0040 e:000039 METHOD <internal:prelude>:149
c:0008 p:0034 s:0035 e:000034 METHOD /usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/lib/github_changelog_generator/options.rb:99
c:0007 p:0084 s:0031 e:000030 METHOD /usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/lib/github_changelog_generator/parser.rb:29
c:0006 p:0010 s:0024 e:000023 METHOD /usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/lib/github_changelog_generator.rb:28 [FINISH]
c:0005 p:---- s:0020 e:000019 CFUNC  :new
c:0004 p:0019 s:0016 e:000015 TOP    /usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/bin/github_changelog_generator:5 [FINISH]
c:0003 p:---- s:0013 e:000012 CFUNC  :load
c:0002 p:0132 s:0008 E:0006e0 EVAL   /usr/local/bundle/bin/github_changelog_generator:23 [FINISH]
c:0001 p:0000 s:0003 E:001820 (none) [FINISH]

-- Ruby level backtrace information ----------------------------------------
/usr/local/bundle/bin/github_changelog_generator:23:in `<main>'
/usr/local/bundle/bin/github_changelog_generator:23:in `load'
/usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/bin/github_changelog_generator:5:in `<top (required)>'
/usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/bin/github_changelog_generator:5:in `new'
/usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/lib/github_changelog_generator.rb:28:in `initialize'
/usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/lib/github_changelog_generator/parser.rb:29:in `parse_options'
/usr/local/bundle/gems/github_changelog_generator-1.15.0.pre.rc/lib/github_changelog_generator/options.rb:99:in `print_options'
<internal:prelude>:149:in `pp'
/usr/local/lib/ruby/2.5.0/delegate.rb:85:in `method_missing'
/usr/local/lib/ruby/2.5.0/delegate.rb:85:in `call'
/usr/local/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/local/lib/ruby/2.5.0/delegate.rb:85:in `method_missing'
/usr/local/lib/ruby/2.5.0/delegate.rb:85:in `call'

-- Machine register context ------------------------------------------------
 RIP: 0x00007f5b9bcb7d70 RBP: 0x00007ffe95896650 RSP: 0x00007ffe95896540
 RAX: 0x0000000000000008 RBX: 0x0000000000000001 RCX: 0x00007f5b9c43ad50
 RDX: 0xffffffffffffffff RDI: 0x0000000000000001 RSI: 0x0000000000000000
  R8: 0x0000000000000048  R9: 0x0000000000000000 R10: 0x0000000000000001
 R11: 0x0000000000000000 R12: 0x0000563e110ca0d0 R13: 0x0000563e1108bd80
 R14: 0x0000000000000000 R15: 0x0000000000000001 EFL: 0x0000000000010202

-- Other runtime information -----------------------------------------------

* Loaded script: /usr/local/bundle/bin/github_changelog_generator

* Loaded features:

    0 enumerator.so
    1 thread.rb
    2 rational.so
    3 complex.so
    4 /usr/local/lib/ruby/2.5.0/x86_64-linux-musl/enc/encdb.so
    5 /usr/local/lib/ruby/2.5.0/x86_64-linux-musl/enc/trans/transdb.so
    6 /usr/local/lib/ruby/2.5.0/x86_64-linux-musl/rbconfig.rb
    7 /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/compatibility.rb
    8 /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/defaults.rb
    9 /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/deprecate.rb
   10 /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/errors.rb
```

**Comment by mokychandrey** [comment]
Plus one. Got same issue.

**Comment by ferrarimarco** [comment]
I updated the `latest` version today, can you please check?

**Comment by ferrarimarco** [comment]
Closing, this is related to the docker image, not to github changelog generator. (and this will be fixed once the 1.15.0 version is out)

**Comment by ferrarimarco** [comment]
See #728
